src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        int numberOfBalls = 3;
+        Lottery[] balls = new Lottery[numberOfBalls];
+        for (int i = 0; i < balls.length; i++) {
+            balls[i] = new Lottery();
+            System.out.println(balls[i].getRandomBall());
+        }
```

Comment:
```suggestion
        Lottery lottery = new Lottery();
        for (int i = 0; i < numberOfBalls; i++) {
            System.out.println(lottery.getRandomBall());
        }
```

Reply:
We don't need an array of lotteries. One object is enough to get new balls from it.